spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :system do
+  describe 'post page' do
+    before(:each) do
+      @user = User.create(
+        name: 'Kanza Tehreen',
+        bio: 'Full Stack Web Developer',
+        photo: 'https://avatars.kanzatehreen.com'
+      )
+      @post = Post.create(title: 'First Post', text: 'This is my first post', author_id: @user.id)
+      @comment = Comment.create(text: 'First comment', author_id: @user.id, post_id: @post.id)
+      Like.create(author_id: @user.id, post_id: @post.id)
+
+      visit(user_posts_path(@user.id))
+    end
+
+    it "shows user's profile picture" do
+      expect(page).to have_css('img[src*="https://avatars.kanzatehreen.com"]')
+    end
```

Comment:
- [ ] Great job with the integration tests 👏. Unfortunately, `shows user's profile picture` test fails. Kindly fix this 🙏 

![image](https://user-images.githubusercontent.com/74814780/215868808-32a94533-2340-4819-97fb-add04fc31f5e.png)

Reply:
Hey @Ugboaja-Uchechi 👋 Thank you for the review. 
But I have a doubt.  Can you please clear this one? All these tests are running in my pc.

**_Here's the screenshot for reference 🔽 That's for all tests_**

![image](https://user-images.githubusercontent.com/106119105/215874199-c6785929-586d-49fe-801f-730858f635fd.png)

**_This is specifically for `post_index_page.rb` 🔽_**

![image](https://user-images.githubusercontent.com/106119105/215874842-49caf967-6e68-4d43-af9f-e5447bdc1910.png)

**_Form my coding partner's system 🔽_**

![image](https://user-images.githubusercontent.com/106119105/215875841-8fb12ff0-862b-4e08-bd8d-aa41610ae507.png)